.github/ISSUE_TEMPLATE/refactor_tickets.md

```diff
@@ -0,0 +1,18 @@
+---
+name: 🐛 Refactor ticket
+about: Help us improve our code by refactoring it.
+labels: refactor
+title: 🐛 [Refactor] 
```

Comment:
```suggestion
title: 🔧 [Refactor] 
```

same here – feel free to choose another one, of course! it was just the first one that came to mind...